astropy/visualization/stretch.py

```diff
@@ -524,3 +537,28 @@ def __call__(self, values, clip=True, out=None):
     def inverse(self):
         """A stretch object that performs the inverse operation."""
         return ContrastBiasStretch(self.contrast, self.bias)
+
+
+class CompositeStretch(CompositeTransform, BaseStretch):
+    """
+    A combination of two stretches.
+
+    Parameters
+    ----------
+    stretch_1 : :class:`astropy.visualization.BaseStretch`
+        The first stretch to apply.
+    stretch_2 : :class:`astropy.visualization.BaseStretch`
+        The second stretch to apply.
+    """
+
+    def __init__(self, stretch_1, stretch_2):
+        super().__init__(stretch_1, stretch_2)
+
+    def __call__(self, values, clip=True, out=None):
+        return self.transform_2(
+            self.transform_1(values, clip=clip, out=out), clip=clip, out=out)
+
+    @property
+    def inverse(self):
+        return CompositeStretch(self.transform_2.inverse,
+                                self.transform_1.inverse)
```

Comment:
For the inverse, I wonder if it would be better to update the implementation in ``CompositeTransform`` to use ``self.__class__`` instead of hard-coding the class name?